python/python/bystro/search/utils/annotation.py

```diff
@@ -39,7 +39,6 @@ class AnnotationOutputs(Struct, frozen=True):
             statistics: Optional[StatisticsOutputs]
                 Basenames of the statistics files, inside the archive
     """
-    output_dir: str
```

Comment:
output_dir on this caused issues during indexing because the bakend does not pass this property, and since we don't need it elsewhere, removing.